Library/Formula/parse-cli.rb

```diff
@@ -0,0 +1,17 @@
+class ParseCli < Formula
+  homepage "https://parse.com"
+  url "https://www.parse.com/downloads/cloud_code/parse",
```

Comment:
For reference, this was [previously rejected](https://github.com/Homebrew/homebrew-binary/pull/138) from homebrew-binary due to the dynamic URL.